.gitignore

```diff
@@ -3,3 +3,5 @@ __pycache__/
 .eggs/
 build/
 *egg-info/
+*.egg
+dist/
```

Comment:
No newline at the end. I don't think it matters to git but it's good practice and avoids git notifying you about it when doing diffs.